### DISCUSSION TITLE
[功能询问] 请问有显示敌人路径的功能吗

### DISCUSSION THREAD

**Discussion by wu5bocheng** [CN-ARC/Mindustry-CN-ARC#87] [Q&A]
我看见敌人生成路径是有相应filter的，请问我们目前有支持显示敌人生成路径吗，如果没有我可以协助开发https://github.com/CN-ARC/Mindustry-CN-ARC/blob/b7d64def12fbd10e1471172ad336f28de1145931/core/assets/bundles/bundle_zh_CN.properties#L568

**Comment by squi2rel** [comment]
还没
